### ISSUE TITLE
train and test video numbers in THUMOS 2014 Dataset

### ISSUE THREAD

**Issue by cc786537662** [yjxiong/action-detection#22]
@yjxiong  hello,thanks for your excellent again,but for now i got  confused about your  thumos14_tag_test_normalized_proposal_list.txt file.

as far as i know, the THUMOS 2014's validation and test set contain 1010 and 1574
untrimmed videos separately. but there are only 20 action categories are involved and annotated temporally.,which is  200 validation set videos and 213 test set videos are used for temporal action detection task,but in your thumos14_tag_test_normalized_proposal_list.txt ,it contains all 1574 videos proposal start time and end time，since it is impossible to train for those videos which does not have annotation information，so during training or testing，did you really train and test on all videos（1010 and 1574） or you just train on（200 +213） and the other videos in  thumos14_tag_test_normalized_proposal_list.txt is not used at all？

**Comment by bityangke** [comment]
SSN are trained on 200  val videos and tested on 211 test videos（2 videos are excluded due to false annotations）

**Comment by sophiazy** [comment]
@bityangke  Could you please tell me which video has false annotations?
